input/cql/Status.cql

```diff
@@ -99,7 +114,12 @@ define fluent function isMedicationOrder(MedicationRequest List<MedicationReques
       and M.intent = 'order'
 
 //Physical Exam, Performed
-define fluent function isPhysicalExamPerformed(Obs List<Choice<"ObservationClinicalResult", "SimpleObservation">>):
```

Comment:
According to the [mapping](https://hl7.org/fhir/us/qicore/STU6/qdm-to-qicore.html#physical-exam-performed), this should be [QI-Core Simple Observation](https://hl7.org/fhir/us/qicore/STU6/StructureDefinition-qicore-simple-observation.html)
[QI-Core Observation Clinical Result](https://hl7.org/fhir/us/qicore/STU6/StructureDefinition-qicore-observation-clinical-result.html) Profile